### PR TITLE
Desktop version 

### PR DESCRIPTION
**Pull request by developertariq**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,86 +4,134 @@
...
-                <a href=""><img src="./icons/Menu.svg"></a>
+                <a id="menu-icon" href=""><img src="./icons/Menu.svg"></a>
+                <ul id="menu-list">
+                    <li><a class="header-menu" href="">Portfolio</a></li>
+                    <li><a class="header-menu" href="">About</a></li>
+                    <li><a class="header-menu" href="">Contact</a></li>
+                    <li><a class="header-menu" href=""><img src="./icons/Mail.svg"></a></li>
+                </ul>                
             </nav>
-            <div id="welcome">
```
Comment (Kamba56) [inline index.html]: - [ ] Consider using an indentation of two spaces in all your documents. This is a convention and also [best practice.](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

**Review by Kamba56 — CHANGES_REQUESTED** [review]
Hi @developertariq, 

Good job so far!!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [x] Great use of Github workflow!👏👏👏
- [x] All linters passed

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (developertariq) [reply index.html]: Hi @Kamba56,
Thanks for your comments.  👍👍👍

**Review by mckent05 — APPROVED** [review]
Hi @developertariq,

# STATUS: APPROVED ✔️ ✔️  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations and well done! 🎉 
approved


## Highlights
- All linter checks are passing ✔️ 
- Your project looks impressive 👏 👏 
- GitHub flow rightly followed 👍 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by developertariq** [comment]
Finally, I have developed a website.

**Pull request merged**